unittesting.json

```diff
@@ -1,11 +1,12 @@
 {
     "deferred": true,
-    "verbosity": 0,
+    "verbosity": 2,
     "capture_console": true,
     "failfast": false,
-    "reload_package_on_testing": true,
+    "reload_package_on_testing": false,
     "start_coverage_after_reload": false,
     "show_reload_progress": false,
     "output": null,
-    "generate_html_report": true
-}
+    "pattern": "test_code_actions.py",
+    "generate_html_report": false
+}
```

Comment:
`"ensure_newline_at_eof_on_save": true` ;)

Reply:
just kidding :)
I am watching this PR, let the CAOS begin :popcorn:

Reply:
But this file is auto-generated by `tests/adjust_unittesting_config_for_ci.py` when running tests the CI-way so not my fault. :)

Reply:
Tell that git blame ;)